pages/news/belugaport.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OPPO watch support
+layout: news
+posted: 2023-04-16
+post: true
+thumbnails: beluga
+summary: The OPPO watch, codenamed "beluga" is now supported by AsteroidOS.
+author: Wannaphong Phatthiyaphaibun (wannaphong)
```

Comment:
I'd love to see  `& Ed Beroset (beroset) ` added to be honest. This was obviously a co-autor and team effort?

Reply:
I wasn't involved in the port, didn't create the pictures and only did a little polishing of the text to make it more idiomatic English, so I'm OK as it is.

Reply:
Thanks for clarifying!